### PR TITLE
Adding links to navbar

### PR DESCRIPTION
**Pull request by benmenachem**
* Added links from navbar to the recruiter' account settings & create new job form page
* Added the major field to the create new job form

### CODE REVIEW

File: SJMaster/templates/base.html [hunk 1]
```diff
@@ -24,9 +24,14 @@
...
                     </li>
                     {% if user.is_authenticated %}
-                    <li class="nav-item">
-                        <a class="nav-link" href="#">Account Settings</a>
+                    {% if is_recruiter %}
+                     <li class="nav-item">
+                        <a class="nav-link" href="recruiter/account_settings/{{ user.id }}">Account Settings</a>
+                    </li>
+                     <li class="nav-item">
+                        <a class="nav-link" href="recruiter/create_new_job_form">Create a new job</a>
```
Comment (guyy13) [inline SJMaster/templates/base.html]: Maybe change it to "Create Job"?

**Review by guyy13 — CHANGES_REQUESTED** [review]

File: SJMaster/templates/base.html (hunk 1 above)
Comment (benmenachem) [reply SJMaster/templates/base.html]: fine

**Review by guyy13 — APPROVED** [review]
lgtm

**Review by guyy13 — APPROVED** [review]

**Review by yaelerdr — APPROVED** [review]

**Review by eldaraharonson — APPROVED** [review]

**Review by ifireball — CHANGES_REQUESTED** [review]
No tests? Am I supposed to guess if this code does what you intended it to do?

**Comment by benmenachem** [comment]
> No tests? Am I supposed to guess if this code does what you intended it to do?

I'll update the PR with tests later today

**Comment by benmenachem** [comment]
> > No tests? Am I supposed to guess if this code does what you intended it to do?
> 
> @ifireball I wasn't really sure about the tests for this PR.
 I consulted with @guyy13 and we agreed that the entire logic of this PR relies on the is_student and is_recruiter model functions that already have tests. therefore I don't think there's anything test here as you instructed not to test the content of the html itself.

**Comment by benmenachem** [comment]
> > No tests? Am I supposed to guess if this code does what you intended it to do?

 @ifireball I wasn't really sure about the tests for this PR. I consulted with @guyy13 and we agreed that the entire logic of this PR relies on the is_student and is_recruiter model functions that already have tests. therefore I don't think there's anything test here as you instructed not to test the content of the html itself.

**Comment by ifireball** [comment]
This PR is not devoid of logic. I think we discussed how to move logic out of templates and into testable locations somewhere already.

Right now there is no test that gives you any form of verification that the right links are displayed under the right conditions.

**Comment by guyy13** [comment]
> This PR is not devoid of logic. I think we discussed how to move logic out of templates and into testable locations somewhere already.
> 
> Right now there is no test that gives you any form of verification that the right links are displayed under the right conditions.

We did discuss it and if I recall correctly you said we'd have to create separate views or even templates in order to move the logic out. Since this is a shared navigation bar across all of the pages I'm not sure how we would do that here. We could try to test the content of the HTML itself in order to check that the links are correct, but again as I recall you said to avoid it. Anyway I really don't understand why we need to test it, we test the context that we pass to the html, if we test that the correct links are showing up we basically test the if statement of Python.

**Comment by ifireball** [comment]
A. Of course you need to test the 'if' statements those commonly get messed up as code is maintained over time.
B. We talked about having different included template files, not different views
C. There is at least one new context updating function in this MR that I don't see tests for either for itself or for the views that use it

**Comment by guyy13** [comment]
> A. Of course you need to test the 'if' statements those commonly get messed up as code is maintained over time.
> B. We talked about having different included template files, not different views
> C. There is at least one new context updating function in this MR that I don't see tests for either for itself or for the views that use it

Ok, we'll try to figure it out

**Pull request closed without merging**